activesupport/lib/active_support/testing/event_reporter_assertions.rb

```diff
@@ -0,0 +1,160 @@
+# typed: true
+# frozen_string_literal: true
+
+module ActiveSupport
+  module Testing
+    module EventReporterAssertions
```

Comment:
```suggestion
    # Provides test helpers for asserting on ActiveSupport::EventReporter notifications.
    module EventReporterAssertions
```

Or something to that effect